packages/now-client/src/deploy.ts

```diff
@@ -23,25 +23,24 @@ async function* createDeployment(
   const debug = createDebug(clientOptions.debug);
   const preparedFiles = prepareFiles(files, clientOptions);
   const apiDeployments = getApiDeploymentsUrl(deploymentOptions);
+  const url = `${apiDeployments}${generateQueryString(clientOptions)}`;
+  const body = JSON.stringify({
+    ...deploymentOptions,
+    files: preparedFiles,
+  });
+
+  debug('Sending deployment creation API request to ' + url);
 
-  debug('Sending deployment creation API request');
   try {
-    const dpl = await fetch(
-      `${apiDeployments}${generateQueryString(clientOptions)}`,
-      clientOptions.token,
-      {
-        method: 'POST',
-        headers: {
-          'Content-Type': 'application/json',
-        },
-        body: JSON.stringify({
-          ...deploymentOptions,
-          files: preparedFiles,
-        }),
-        apiUrl: clientOptions.apiUrl,
-        userAgent: clientOptions.userAgent,
-      }
-    );
+    const dpl = await fetch(url, clientOptions.token, {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: body,
```

Comment:
```suggestion
      body,
```

Reply:
Why is `body` a variable now? It's not used anywhere else

Reply:
The changes up here were to add logging. But I removed the log for body because it could be very verbose depending on how many files there are.